politeiad/backendv2/tstorebe/tstore/tstore.go

```diff
@@ -182,7 +181,7 @@ func (t *Tstore) Fsck(allTokens [][]byte) error {
 
 		err := p.client.Fsck(allTokens)
 		if err != nil {
-			return errors.Errorf("plugin %v fsck: %v",
+			return fmt.Errorf("plugin %v fsck: %v",
```

Comment:
Why using `fmt`?

Reply:
See #1428 for details on why we don't use `fmt.Errorf` anymore.